highfidelity/hifi
# File dialog crash qt59

Test plan:
- Build interface with Qt 5.9
- Run application and press Ctrl-J
- Press FROM DISK button
- walk thru different folders, like scripts-archive/examples etc
- make sure no crash occurs
- do this in both Tablet and HMI modes
